frontend/src/global_styles/content/_widget_box.sass

```diff
@@ -81,6 +81,7 @@ $widget-box--enumeration-width: 20px
 
     &--teaser-image
       width: 200px
+      margin: auto
```

Comment:
All of the changes made in this file affect not only the upsale block, but also the image in the "What's new" block. I strongly assume, that we don't want that.

Reply:
Thanks @HDinger for your comments :+1: 
I wanted to make the picture stays in the middle of the block when the screen size changes, like other places that we have an enterprise banner. I checked the new feature component too, I thought that it won't make it looks bad.

How do you think is better to set the style for the image used in banners and home page upsale block? Is it ok to remove the `margin: auto` style for enterprise banner too?

Reply:
At least there is a clear visual difference between the two options, the image is smaller in your version and placed differently:

**With your changes**
<img width="250" alt="Bildschirmfoto 2022-08-09 um 08 30 36" src="https://user-images.githubusercontent.com/7457313/183580510-f998674b-0b3d-4019-8633-7b39f35dd93d.png">


**Without your changes**
<img width="250" alt="Bildschirmfoto 2022-08-09 um 08 32 51" src="https://user-images.githubusercontent.com/7457313/183580519-9c709b87-0b16-483a-9a17-cab6e73af999.png">

I am not saying that one version looks better than the other, but it was a decision by the product team a while ago to place the image at the top, so I'd keep it that way.
So I'd not touch the global widget_box.sass file but move the changes to the `enterprise.sass` where we touch the styling of that home screen block already anyway.